src/components/Workspace.jsx

```diff
@@ -231,6 +234,10 @@ class Workspace extends React.Component {
     this.props.dispatch(toggleDashboard());
   }
 
+  _handleEditorsUpdateVerticalFlex(data) {
+    this.props.dispatch(editorsUpdateVerticalFlex(data));
+  }
```

Comment:
This kind of thing (while 100% appropriate for the way Popcode’s components are currently structured) definitely feels like a strong argument for sprinkling more connected containers throughout the hierarchy.